native/src/modules/common/components/ProgressBar.js

```diff
@@ -0,0 +1,74 @@
+// @flow
```

Comment:
I think this is rather a `ProgressSpinner` as the bar will be implemented separately, right?

Reply:
done